backend/src/routes/BankingRoutes.ts

```diff
@@ -3,23 +3,24 @@ import { IReq, IRes } from './types/express/misc';
 import { BankingRoutesLinkData } from '@src/Context';
 
 
+
 require('dotenv').config();
 const { Configuration, PlaidApi, Products, PlaidEnvironments} = require('plaid');
 
 
 const PLAID_CLIENT_ID = process.env.PLAID_CLIENT_ID;
 const PLAID_SECRET = process.env.PLAID_SECRET;
-const PLAID_ENV = 'sandbox';
+const PLAID_ENV =  'sandbox';
 
 // PLAID_PRODUCTS is a comma-separated list of products to use when initializing
 // Link. Note that this list must contain 'assets' in order for the app to be
 // able to create and retrieve asset reports.
-const PLAID_PRODUCTS = ['auth,transactions'];
+const PLAID_PRODUCTS = ["auth","transactions"];
```

Comment:
Did you run prettier? I'm trying to understand why we went from single quotes to double-quotes?

Reply:
no, I forgot